dotnet/src/Connectors/Connectors.Amazon/Bedrock/Core/Models/Meta/MetaService.cs

```diff
@@ -75,7 +75,7 @@ public ConverseRequest GetConverseRequest(string modelId, ChatHistory chatHistor
     }
 
     /// <inheritdoc/>
-    public IEnumerable<string> GetTextStreamOutput(JsonNode chunk)
+    public IEnumerable<string> GetTextStreamOutput(string modelId, JsonNode chunk)
```

Comment:
> [!NOTE]
> Apply to all services

Revert this change as all the services were designed to not use the `modelId` information, worth having a specific service to handle the different `Nova` requests.


```suggestion
    public IEnumerable<string> GetTextStreamOutput(JsonNode chunk)
```

Reply:
done